code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm

```diff
@@ -136,7 +136,7 @@
 /obj/item/weapon/robot_module/robot/knine
 	name = "k9 robot module"
 	sprites = list(
-					"K9 hound" = "k9",
+					"K9 hound" = "stray",
```

Comment:
...I think this was a test change that somehow made it in?

Reply:
...big whoops. Gonna fix that